torch/csrc/jit/codegen/cuda/lower_predicate_elimination.cpp

```diff
@@ -23,6 +23,12 @@ namespace {
 //   predicating these ops will require extra steps to ensure that
 //   the whole warp will get the same value.
 void assertOnWarpOps(const Expr* expr) {
+  if (auto uop = dynamic_cast<const UnaryOp*>(expr)) {
+    TORCH_INTERNAL_ASSERT(
+        !ir_utils::isLdMatrixOp(expr),
+        "Predicate elimination: cannot eliminate pred for ldmatrix, use exact parallel dims",
+        expr);
```

Comment:
Just curious are we doing this for all error messages now? This would work if `<<` operator is overloaded and I kind of noticed that it is in `ir_printer` that not every file included.

Reply:
We used to rely on `operator<<` overloaded for pointers, which wasn't very robust (at least for me). Explicit `toString()` should be used now.

Reply:
Yes it'd need to always include ir_printer.h to work. It seems to be more convenient to use `toString`, I can update. Just want to be sure about what pattern.  we want to stick with.

Reply:
updated. resolving.